src/decoration.c

```diff
@@ -41,7 +41,7 @@
 
 #define PLACE_DECORATION_SELECTOR_TAG 0xbe5
 #define PLACE_DECORATION_PLAYER_TAG   0x008
-#define NUM_DECORATION_FLAGS (FLAG_DECORATION_14 - FLAG_DECORATION_1 + 1)
+#define NUM_DECORATION_FLAGS (FLAG_DECORATION_13 - FLAG_DECORATION_1 + 1)
```

Comment:
Yes; note this is only for sprite decorations, like dolls and cushions. If I remember right it's pretty hard to actually have that many sprite decorations, though, since they have to be placed on mats or desks.

I'd prefer keeping the flag and adding a comment marking it unused in `flags.h`

Reply:
If we mark it as unused we need to specify that it can't be used, not that it's unused in vanilla but free to use (as is the case for all other unused flags and variables)

Reply:
We can't quite reduce this, as the flag is still used across multiple instances.
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/04e8c3d5-f58a-496e-94d5-6d93bceeca18)
Plus, changing `NUM_DECORATION_FLAGS` does stuff like this:
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/591ed819-a532-4693-b29b-ac469fe97b6e) ![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/e2199365-c165-4761-80f6-901996f22f03)

However, merrp is right it doesn't seem possible to have more than 13 sprite decorations, but thankfully that give us the exact amount of space needed for the follower without needing to modify the logic or `NUM_DECORATION_FLAGS`.
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/57caa479-275c-4e4f-aeb7-e044a09c543d)